src/data/cryptocurrencies.js

```diff
@@ -78,6 +78,14 @@ const cryptocurrenciesArray: CryptoCurrency[] = [
     color: "#ffae35",
     units: ethereumUnits
   },
+  {
+    id: "ethereum_classic",
+    name: "Ethereum Classic",
+    ticker: "ETC",
+    scheme: "ethereum",
+    color: "#ffae35",
+    units: ethereumUnits
```

Comment:
needs to be another list, where the first item will be `code: "ETC"`

also, check with designers what will be ETC color (if defined).

finally, I think we need a different scheme, maybe for now you can put "ethereumclassic" in scheme until we really figure it out.. i think in the future, we will rethink this scheme thing, because generating QRCode with this won't be supported everywhere maybe (like third party app that scan this.. maybe we should allow no scheme & fallback on diff string)